react-datetime.d.ts

```diff
@@ -106,6 +106,18 @@ declare module ReactDatetime {
      When true, once the day has been selected, the react-datetime will be automatically closed.
      */
     closeOnSelect?: boolean;
+
+    /*
+    Allow to add some constraints to the time selector. It accepts an object with the format {hours:{ min: 9, max: 15, step:2}}
```

Comment:
It seems like the styling of this block isn't correct, if you compare it with the other comment blocks. It should be pushed to the right a bit more. Also, please break the line manually so it doesn't have to break on it's own (look how long the lines are in other parts of this file). :)